### DISCUSSION TITLE
Table Row-level policies to leverage API calls

### DISCUSSION THREAD

**Discussion by yashwanth2804** [planetscale/discussion#172] [Feature Requests] [answered]
In Supabase there is row level policy , which is a nice feature because we could directly call from client with client-id and check for row level access, Is there something in planetscale already available ?
In order to use splanetscale do need too employ a API server ?
Thank you.

**Comment by jgreet** [comment, marked as answer]
The answer is https://github.com/planetscale/beta/discussions/157 is still current.

There isn't an equivalent row level security feature in PlanetScale or MySQL.

You could use an API server, or a serverless platform that allows for database connections like Vercel.
